runtime/convertValues.go

```diff
@@ -713,13 +713,17 @@ func exportPathCapabilityValue(
 	v *interpreter.PathCapabilityValue, //nolint:staticcheck
 	inter *interpreter.Interpreter,
 ) (cadence.Capability, error) {
-	borrowType := inter.MustConvertStaticToSemaType(v.BorrowType)
-	exportedBorrowType := ExportMeteredType(inter, borrowType, map[sema.TypeID]cadence.Type{})
+	var exportedBorrowType cadence.Type
+
+	if v.BorrowType != nil {
```

Comment:
Do we need to do the same for `IDCapabilityValue` above?

Reply:
Not sure 🤔 I assumed that all ID capabilities have a type now, but maybe that's not actually the case?